hyp3_metadata/templates/insar/readme.md.txt.j2

```diff
@@ -0,0 +1,298 @@
+
+***************
+# Product Contents #
+
+The files generated in this process include:
+
+1. Wrapped Interferogram (PNG images, KMZ file, GeoTIFF (*optional*))
+2. Unwrapped Interferogram (GeoTIFF, PNG images, KMZ file)
+3. Line-of-Sight Displacement Map (GeoTIFF) - *Optional*
+4. Vertical Displacement Map (GeoTIFF)
+5. Coherence Map (GeoTIFF)
+6. Amplitude Image (GeoTIFF)
+7. Parameter Documentation (Text File)
+8. Look Vector maps (GeoTIFFs) - *Optional*
+9. Incidence Angle map (GeoTIFF) - *Optional*
+10. DEM used to process the data (GeoTIFF) - *Optional*
+11. Water Mask (GeoTIFF) - *Optional*
+
+*See below for detailed descriptions of each of the product files.*
+
+----------------
+## 1. Wrapped Interferogram
+
+The wrapped interferogram is output in both a georeferenced PNG and KMZ format by default, and the full GeoTIFF file can also be optionally included. Because the fringes are on a wrapped 2-pi scale rather than representing continuous values, a wrapped interferogram tends to be most useful as a color visualization. This product displays deformation in multiples of half of the sensor wavelength, so areas with fringes that are very close together indicate greater amounts of deformation in the line-of-sight direction from the sensor.
+
+The wrapped interferogram image is generated from the interferogram phase prior to unwrapping. The PNG and KMZ files are tagged with _color_phase, and are 2048 pixels wide. The optional GeoTIFF is tagged with _wrapped_phase.tif
+
+*The GeoTIFF version of the wrapped interferogram is optional. Select the "Include Wrapped GeoTIFF" option in the On Demand HyP3 Processing Options to include it in the product package.*
+
+----------------
+## 2. Unwrapped Interferogram
+
+The unwrapped interferogram uses a reference point to convert the wrapped 2-pi scale into a continuous scale (multiples of 2 pi radians), and is the most important and complicated step in InSAR processing. It relies on having a reference point beyond the area of deformation, and is most effective when there are not any areas with deformation greater than half the sensor wavelength within the space of a single fringe. In regions of large-magnitude deformations or extremely steep topography, the fringes may be too close together to resolve, or one fringe may actually represent a jump of multiple phase wraps. Gamma software uses algorithms to detect phase inconsistencies and treat them appropriately, but some deformations may still be too large to generate reliable data.
```

Comment:
`Gamma` should be stylized `GAMMA`, I believe
https://www.gamma-rs.ch/software

Reply:
Corrected